ob_v2p0/extensions/assessmentExtension/index.html

```diff
@@ -7,7 +7,7 @@
     var respecConfig = {
       specTitle: "Open Badges Assessment Extension",
       shortName: "ob-assessment",
-      specStatus: "IMS Candidate Final",
+      specStatus: "IMS Candidate Final Public",
       specDate: "April 12th, 2018",
```

Comment:
Should `specDate` be changed to October 7, 2020? Revision History explains history of each date change.

Reply:
Yes, thank you. specDate should match revision history.